crates/chia-consensus/src/fast_forward.rs

```diff
@@ -1,43 +1,16 @@
 use crate::error::{Error, Result};
 use chia_protocol::Bytes32;
 use chia_protocol::Coin;
+use chia_wallet::singleton::SingletonArgs;
+use chia_wallet::singleton::SingletonSolution;
+use chia_wallet::singleton::SingletonStruct;
 use chia_wallet::singleton::SINGLETON_TOP_LAYER_PUZZLE_HASH;
```

Comment:
```suggestion
use chia_wallet::singleton::{SingletonArgs, SingletonSolution, SingletonStruct, SINGLETON_TOP_LAYER_PUZZLE_HASH};
```

Reply:
Done, it formats to multiple lines though